inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -24,20 +25,142 @@ namespace {
         return config;
     }
 
+    std::string GetNetworkPrecision(const InferenceEngine::CNNNetwork &network) {
+        for (auto&& layer : network.getInputsInfo()) {
+            auto precision = layer.second->getPrecision();
+            auto name = std::string(precision.name());
+            if (name == "I8") {
+                name = "INT8";
+            }
+            return name;
+        }
+        return {};
+    }
```

Comment:
move comment from #5549:
ilya: this is not network precision. It's precision of input layer..
shoujiang: well, I remembered Maxim provided some code like:
auto nGraphFunc = network.getFunction();
for (auto &node : nGraphFunc->get_ordered_ops()) {
// auto type0 = node->input_value(0).get_element_type(); //input
auto type1 = node->input_value(1).get_element_type(); //weights
}
so is that expected? should we use typ1 or type0 as the network precision? there are lots of ops, should we use only the first one? thanks!
ilya: int8 network - if we have FakeQuantize operations in the graph
fp16 network - if we have convolution weights stored in fp16
fp32 network - if we have convolution weights stored in fp32